testsuite/tests/templates/basic/bad_instantiate_repeated_param.reference

```diff
@@ -0,0 +1,4 @@
+File "_none_", line 1:
+Error: Float_list_element and Int_list_element
+       were both compiled with -as-argument-for List_element.
+       Only one argument may be given for each parameter.
```

Comment:
> Error: Cannot supply both Float_list_element and Int_list_element
       as arguments for the parameter List_element.
    Hint: Both were compiled with `-as-argument-for List_element`.